internal/apis/contour/v1alpha1/doc.go

```diff
@@ -0,0 +1,18 @@
+// Copyright © 2018 Heptio
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
as I mentioned in the call, maybe we bite the bullet and move this to the top level, ie:

github.com/heptio/contour/apis/contour/v1alpha1/...

Reply:
I can move it out